config/routes.rb

```diff
@@ -11,15 +11,26 @@
     end
   end
 
-  resources :organizations, only: [:index, :show, :edit, :create, :update, :new]
+  resources :organizations, only: [:index, :show, :edit, :create, :update, :new] do
+    resources :memberships do
```

Comment:
It's probably more concise. Because the Membership class acts as a bridge between Organization and User rather than a more traditional resource with typical actions, treating it as such in the routes makes sense. This won't be the case if in the future Membership class is expanded to include linkages between classes other than Organization and User.